Automattic/wp-calypso
# Add auto language direction in reader

This pull request addresses the issue of people using calypso in LTR interface ( English )
and reading blogs in RTL language ( such as Hebrew )


|  Before | After  |
|---|---|
| ![screen shot 2016-11-28 at 15 52 52](https://cloud.githubusercontent.com/assets/326402/20670762/2a21b0d8-b583-11e6-9cd5-f62ac1da0f5c.png)  | ![screen shot 2016-11-28 at 15 18 36](https://cloud.githubusercontent.com/assets/326402/20670774/3d1fb3e2-b583-11e6-89c8-5015c7040b67.png)  |
| ![screen shot 2016-11-28 at 15 57 39](https://cloud.githubusercontent.com/assets/326402/20670824/75df689e-b583-11e6-8287-5c84936ccd88.png) | ![screen shot 2016-11-28 at 15 26 01](https://cloud.githubusercontent.com/assets/326402/20670838/82447ad4-b583-11e6-81d6-f99b95bdd8d8.png)  |
| ![screen shot 2016-11-28 at 15 59 53](https://cloud.githubusercontent.com/assets/326402/20670896/cf9ab190-b583-11e6-8b60-5d8a3f35ffea.png)  | ![screen shot 2016-11-28 at 16 00 10](https://cloud.githubusercontent.com/assets/326402/20670904/db3e2a18-b583-11e6-8e1f-74d8af10638f.png)  |

#### Testing instructions
  
1. Run `git checkout add/auto-direction` and start your server, or open a [live branch](https://calypso.live/?branch=add/auto-direction)
2. Open the [`Home` page](http://calypso.localhost:3000/)
3. Visit reader's following stream and assert you see the posts in the correct directionality
4. Visit reader's full post view and assert you see the post and comments in the correct directionality
5. Start typing in the comments textarea, it should change directionality according to what you type ( RTL / LTR lang )
  
#### Reviews
  
   
@Automattic/sdev-feed
/cc @yoavf @bgbg @ranh @danieldanilov
